drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -209,6 +209,30 @@
     opacity: 0.80;
 }
 
+
+/* Horizontal scroll for item bank on mobile */
+.chromeless .xblock--drag-and-drop .target {
+    display: block;
+    overflow: auto;
+}
+
+.chromeless .xblock--drag-and-drop .drag-container .option {
```

Comment:
This needs to be scoped to only items in the item bank container, otherwise it breaks positioning of items placed in drop zones as well as items actively being dragged. Replace with: `.chromeless .xblock--drag-and-drop .drag-container .item-bank .option { ... }`.